Bottom: b790fbc23eb3c31cdc29166ace11bb24253cf577
Top:    f63888d0372e86697d078aa5425f992e2d0ed397
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-20 15:55:07 +0100

Refresh of hid-gtk-some-noop-changes-to-t

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 5470438..be922a6 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1370,25 +1370,25 @@ SwapSides (int argc, char **argv, Coord x, Coord y)
   if (argc > 0)
     {
       switch (argv[0][0]) {
-      case 'h':
-      case 'H':
-        ghid_flip_x = ! ghid_flip_x;
-        do_flip_x = 1;
-        break;
-      case 'v':
-      case 'V':
-        ghid_flip_y = ! ghid_flip_y;
-        do_flip_y = 1;
-        break;
-      case 'r':
-      case 'R':
-        ghid_flip_x = ! ghid_flip_x;
-        ghid_flip_y = ! ghid_flip_y;
-        do_flip_x = 1;
-        do_flip_y = 1;
-        break;
-      default:
-        return 1;
+        case 'h':
+        case 'H':
+          ghid_flip_x = ! ghid_flip_x;
+          do_flip_x = 1;
+          break;
+        case 'v':
+        case 'V':
+          ghid_flip_y = ! ghid_flip_y;
+          do_flip_y = 1;
+          break;
+        case 'r':
+        case 'R':
+          ghid_flip_x = ! ghid_flip_x;
+          ghid_flip_y = ! ghid_flip_y;
+          do_flip_x = 1;
+          do_flip_y = 1;
+          break;
+        default:
+          return 1;
       }
       /* SwapSides will swap this */
       Settings.ShowSolderSide = (ghid_flip_x == ghid_flip_y);
